benchmarks/DeepLearning/Ops/RFFTOp/CMakeLists.txt

```diff
@@ -0,0 +1,50 @@
+cmake_minimum_required(VERSION 3.10)
+project(RFFTBenchmark)
+
+add_custom_command(
+  OUTPUT
+    ${CMAKE_CURRENT_SOURCE_DIR}/result.txt
+  COMMAND
+    python3 ${CMAKE_CURRENT_SOURCE_DIR}/RFFT.py ${CMAKE_CURRENT_SOURCE_DIR}
+  COMMENT "Generating result.txt"
+)
+
+add_custom_target(generate_result_txt ALL
+  DEPENDS ${CMAKE_CURRENT_SOURCE_DIR}/result.txt
+  COMMENT "Ensure result.txt is generated"
+)
+
+# Define variables for the cross-compilation toolchain and options.
+if (CROSS_COMPILE_RVV)
+  set(RISCV_GNU_TOOLCHAIN ${BUDDY_MLIR_BUILD_DIR}/thirdparty/riscv-gnu-toolchain)
+  set(RISCV_GNU_TOOLCHAIN_SYSROOT ${RISCV_GNU_TOOLCHAIN}/sysroot)
+  set(BUDDY_OPT_TRIPLE riscv64)
+  set(BUDDY_OPT_ATTR +v,+m)
+endif()
+
+set(LLVM_MLIR_BINARY_DIR ${BUDDY_MLIR_BUILD_DIR}/../llvm/build/bin)
```

Comment:
`set(LLVM_MLIR_BINARY_DIR ${BUDDY_MLIR_BUILD_DIR}/../llvm/build/bin)` 
This variable already defined in `buddy-benchmark/CMakeLists.txt`  line 54.  
Please remove the variable declaration at line 25.